### DISCUSSION TITLE
Any plans for supporting server-side rendering?

### DISCUSSION THREAD

**Discussion by hiql** [patternfly/patternfly-react#9683] [General]
Most developers choose NextJS/Remix as their react framework, nowadays...

**Comment by Venefilyn** [comment]
@nicolethoen Hi! Just checking up if there's been any discussions about this

**Comment by nicolethoen** [comment]
hello - there has not been more talk about this on the PatternFly team proper. 
Formally, we support the technology stack of the Red Hat product portfolio, so those products are not moving to that stack for their react framework at this time.

What sorts of changes from PF would be needed to support NextJS/Remix?
